sample/refresh_token_sample.py

```diff
@@ -48,15 +48,15 @@ def turn_on_logging():
     RESOURCE, 
     sample_parameters['username'],
     sample_parameters['password'],
-    sample_parameters['clientid'])
+    sample_parameters['clientId'])
```

Comment:
Sounds good. Avoid breaking change weighs more than just for consistency with other samples